libr/core/cmd_anal.c

```diff
@@ -379,26 +379,32 @@ static const char *help_msg_afvs[] = {
 };
 
 static const char *help_msg_ag[] = {
-	"Usage:", "ag[?f]", " Graphviz/graph code",
-	"ag", " [addr]", "output graphviz code (bb at addr and children)",
-	"ag-", "", "Reset the current ASCII art graph (see agn, age, agg?)",
-	"aga", " [addr]", "idem, but only addresses",
-	"agr", "[j] [addr]", "output graphviz call graph of function",
-	"agg", "", "display current graph created with agn and age (see also ag-)",
-	"agc", "[*j] [addr]", "output graphviz call graph of function",
-	"agC", "[j]", "Same as agc -1. full program callgraph",
-	"agd", " [fcn name]", "output graphviz code of diffed function",
-	"age", "[?] title1 title2", "Add an edge to the current graph",
-	"agf", " [addr]", "Show ASCII art graph of given function",
-	"agg", "[?] [kdi*]", "Print graph in ASCII-Art, graphviz, k=v, r2 or visual",
-	"agj", " [addr]", "idem, but in JSON format",
-	"agJ", " [addr]", "idem, but in JSON format with formatted disassembly (like pdJ)",
-	"agk", " [addr]", "idem, but in SDB key-value format",
-	"agl", " [fcn name]", "output graphviz code using meta-data",
-	"agn", "[?] title body", "Add a node to the current graph",
-	"ags", " [addr]", "output simple graphviz call graph of function (only bb offset)",
-	"agt", " [addr]", "find paths from current offset to given address",
-	"agv", "", "Show function graph in web/png (see graph.web and cmd.graph) or agf for asciiart",
+	"Usage:", "ag<graphtype><format> [addr]", "",
+	"Graph types:", "", "",
+	"c", " [fcn addr]", "Function callgraph",
```

Comment:
you shuold put the whole command in here, otherwise it will break ?* to enumerate all recursive commands